### PR TITLE
[READY-TO-REVIEW] Update V1_cognitive_app.md

### PR DESCRIPTION
**Pull request by jarusso**

### CODE REVIEW

**Comment by jarusso** [comment]
@miguelestrada review and merge please

File: guides/V1_cognitive_app.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ is: 'published'
...
+The Make it Cognitive section of your App's dashboard allows you to associate an instance (workspace) of [Watson Assistant](https://www.ibm.com/watson/developercloud/conversation.html) with your application. When you App is installed to a space, new messages will be sent to your Watson Assistant instance by Watson Work Services using the credentials you've provided in the Make it Cognitive section of the App's dashboard.
 
-In your App's dashboard, you provide a Watson Conversation ID. This ID is found in [Watson Conversation](https://www.ibmwatsonconversation.com) by clicking View Details from the menu on your Watson Conversation Workspace.
+In your App's dashboard, you provide a Watson Assistant ID. This ID is found in [Watson Assistant](https://www.ibmwatsonconversation.com) by clicking View Details from the menu on your Watson Assistant Workspace.
```
Comment (miguelestrada) [inline guides/V1_cognitive_app.md]: Replace by adding "Workspace " to qualify the ID.... It is not a Watson Assistant ID, but rather a Watson Assistant Workspace ID

`In your App's dashboard, you provide a Watson Assistant Workspace ID. This ID is found in [Watson Assistant](https://www.ibmwatsonconversation.com) by clicking View Details from the menu on your Watson Assistant Workspace.`

**Comment by miguelestrada** [comment]
@jarusso see simple edit to clarify the reference to ID

**Comment by jarusso** [comment]
good catch - done @miguelestrada

**Comment by jarusso** [comment]
@miguelestrada bump on this - please review and merge

**Pull request merged**